tensorflow/buildspec.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Would this file be merged to master as it is, or, will it be copied to `buildspec-2.6.yml` and then merged to master?
If it will be copied to `buildspec-2.6.yml`, can you make that change as well?

Reply:
This wont be merged to master and will be reverted. I'll create separate PR to change patch branch tf2.6 to build inference